ci-operator/step-registry/baremetal/lab/ipi/conf/virtual-media/baremetal-lab-ipi-conf-virtual-media-commands.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "${REDFISH_VIRTUAL_MEDIA:-false}" != "true" ]; then
+  echo "Redfish virtual media is not enabled. Skipping..."
+  exit 0
+fi
+
+if [ "${ENABLE_PROVISIONING_NETWORK:-true}" != "true" ]; then
```

Comment:
Is there a case where we use virtual media and the provisioning network is enabled yet?

Reply:
Actually no. virtual media is needed when provisioning network is disabled. Removing